tests_recording/test_api.py

```diff
@@ -170,31 +141,14 @@ def test_changelog_sync(self):
         new_downstream_spec_content = self.api.dg.absolute_specfile_path.read_text()
         assert changed_upstream_spec_content == new_downstream_spec_content
 
-    @unittest.skipIf(
-        int(getattr(rebasehelper, "VERSION", "0.18.0").split(".")[1]) >= 24,
-        "Old version of RebaseHelper raises exception",
-    )
     def test_version_change_exception(self):
         """
         check if it raises exception, because sources are not uploaded in distgit
         Downgrade rebasehelper to version < 0.24.0
         """
-        self.assertRaises(RebaseHelperError, self.check_version_increase)
-
-    @unittest.skipIf(
-        int(getattr(rebasehelper, "VERSION", "0.18.0").split(".")[1]) < 24,
-        "New version of rebasehelper >=24",
-    )
-    @unittest.skip("Recordings are broken")
-    def test_version_change_exception_new_rebase_helper(self):
-        """
-        check if it raises exception, because sources are not uploaded in distgit
-        New version raises same error, but it raises it on another place
-        Upgrade rebase helper >= 0.24.0
-        """
-        self.assertRaises(RebaseHelperError, self.check_version_increase)
+        self.assertRaises(GithubAPIException, self.check_version_increase)
 
-    @unittest.skip("Recordings are broken")
+    @unittest.skip("This is broken, because have to flexmock anther part ")
```

Comment:
what would be a better solution here than skipping the test?

Reply:
Sure, I'll fix it, in some next PR, because it will need flexmock another part. So nothong generic like this PR.